scripts/deps.mk

```diff
@@ -12,14 +14,16 @@ check_$(1):
 	 fi
 endef
 
-# check the existence of a specific python library
-# usage: check_dep_pylib <library name>
+# usage: check_dep_pylib <library name> <variable>
+#
+# Create a target that checks the existence of the specified python 3 library, and
+# append that target to the given variable.
 define check_dep_pylib =
```

Comment:
This function is never called because you use `check_dep_py3lib` above. You should correct the name of the function (or fix the invocation in `scripts/kconfig/kconfig.mk`

Reply:
Thanks for pointing it out. I'll fix the definition.